ui-tests/src/test/java/org/openmrs/reference/EditFormTest.java

```diff
@@ -62,10 +63,9 @@ public void editFormTest() throws Exception {
 
     @After
     public void tearDown() throws Exception {
-	if(headerPage !=null) {
-        headerPage.clickOnHomeIcon();
-        headerPage.logOut();
+        if (headerPage != null) {
```

Comment:
Are sure you need this ? What would happen when not added?

Reply:
For all these form test related tasks we need to handle nullpointer exceptions that is thrown with in header page, if not added the test will fail with NullPointer Exception